itools-packager/src/main/java/com/powsybl/itools/ItoolsPackagerMojo.java

```diff
@@ -194,9 +288,16 @@ public void execute() {
             Files.createDirectories(libDir);
             copyFiles(copyToLib, libDir);
 
-            getLog().info("Zip package");
+            // create package
             String archiveNameNotNull = archiveName != null ? archiveName : packageNameNotNull;
-            zip(packageDir, targetDir, targetDir.resolve(archiveNameNotNull + ".zip"));
+            int lastIndexOf = archiveNameNotNull.lastIndexOf(".");
+            if (lastIndexOf < 0) {
+                // file extension: zip by default
+                archiveNameNotNull += ".zip";
+            }
```

Comment:
You do not need this change: a similar test is done in the `createPackage` function.

Reply:
This code is needed to create an archive name with the zip extension.
For example, archiveName is from namePackage ("powsybl" without extension).
I moved this code in createPackage.